src/open_inwoner/plans/tests/test_views.py

```diff
@@ -44,7 +44,11 @@ def setUp(self) -> None:
 
         self.login_url = reverse("login")
         self.list_url = reverse("collaborate:plan_list")
-        self.create_url = reverse("collaborate:plan_create")
+        self.choose_template_url = reverse("collaborate:plan_choose_template")
+        # TODO: new tests for choosing template form
+        self.create_url = reverse("collaborate:plan_create_no_template")
+        self.create_with_template_url = reverse("collaborate:plan_create_with_template")
```

Comment:
```suggestion
        self.create_from_template_url = reverse("collaborate:plan_create_with_template")
```
Same as above. Look at your comment: you say "creating plan from template", that's the natural way to express this, and should be reflected in the naming